Tests/subset/subset_test.py

```diff
@@ -1352,5 +1352,55 @@ def test_subset_svg_missing_lxml(ttf_path):
         subset.main([str(ttf_path), "--gids=0,1"])
 
 
+def test_subset_COLR_glyph_closure(tmp_path):
+    # https://github.com/fonttools/fonttools/issues/2461
+    font = TTFont()
+    ttx = pathlib.Path(__file__).parent / "data" / "BungeeColor-Regular.ttx"
+    font.importXML(ttx)
+
+
+    color_layers = font["COLR"].ColorLayers
+    assert ".notdef" in color_layers
+    assert "Agrave" in color_layers
+    assert "grave" in color_layers
+
+    font_path = tmp_path / "BungeeColor-Regular.ttf"
+    subset_path = font_path.with_suffix(".subset.ttf)")
+    font.save(font_path)
+
+    subset.main(
+        [
+            str(font_path),
+            "--glyph-names",
+            f"--output-file={subset_path}",
+            "--glyphs=Agrave",
+        ]
+    )
+    subset_font = TTFont(subset_path)
+
+    glyph_order = subset_font.getGlyphOrder()
+
+    assert glyph_order == [
+        ".notdef",  # '.notdef' is always included automatically
+        "A",
+        "grave",  # included because glyf component of Agrave
+        "Agrave",
+        ".notdef.alt001",
+        ".notdef.alt002",
+        "A.alt002",
+        "Agrave.alt001",
+        "Agrave.alt002",
+        "grave.alt002",
+    ]
+
+    color_layers = subset_font["COLR"].ColorLayers
+    assert ".notdef" in color_layers
+    assert "Agrave" in color_layers
+    # 'grave' not included in COLR because absent after COLR glyph closure;
+    # only added in subset font after subsequent glyf closure.
+    assert "grave" not in color_layers
```

Comment:
Suggested clarification

# Agrave 'glyf' uses grave. It should be retained in 'glyf' but NOT in COLR when we subset down to Agrave.

Reply:
added your comment